### PR TITLE
Selectable language for suggestion labels 

### PR DESCRIPTION
**Pull request by juhoinkinen**
Adds a menu for selecting the language of the labels of the suggestions. Default selection is "text language", in which case the labels are not changed, but the ones coming from http://ai.finto.fi (Annif) are retained. If a specific language is selected, then the URIs of the suggestions are used to get the labels from http://api.finto.fi with the selected language. If something fails in the query, an alert window is shown.

### CODE REVIEW

**Comment by juhoinkinen** [comment]
At least this case remains to be solved: if there is no term for a concept in a language, like for [Pohjanmaa in English](https://finto.fi/yso-paikat/en/page/?clang=en&uri=p94344), that label is not shown in the list of suggestions (or an empty string is shown):

image

File: ai.finto.fi/static/js/i18n.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ jQuery(function() {
             "nav-subject-indexing": "Subject indexing",
             "project": "Vocabulary and text language",
             "limit": "Maximum # of suggestions",
+            "label-language": "Labels language",
```
Comment (osma) [inline ai.finto.fi/static/js/i18n.js:53]: "Term language" could be better?

File: ai.finto.fi/static/js/i18n.js [hunk 2]
```diff
@@ -50,6 +50,11 @@ jQuery(function() {
...
+            "label-language-option-project": "project",
```
Comment (osma) [inline ai.finto.fi/static/js/i18n.js]: "text language" ?

File: ai.finto.fi/static/js/i18n.js [hunk 3]
```diff
@@ -72,6 +77,11 @@ jQuery(function() {
             "nav-subject-indexing": "Innehållsbeskrivning",
             "project": "Vokabulär och textens språk",
             "limit": "Maximalt antal förslag",
+            "label-language": "Förslag språk",
```
Comment (osma) [inline ai.finto.fi/static/js/i18n.js:80]: "Termernas språk"

File: ai.finto.fi/static/js/i18n.js [hunk 4]
```diff
@@ -72,6 +77,11 @@ jQuery(function() {
...
+            "label-language-option-project": "projekt",
```
Comment (osma) [inline ai.finto.fi/static/js/i18n.js]: "textens språk"

File: ai.finto.fi/static/js/i18n.js [hunk 5]
```diff
@@ -95,6 +105,11 @@ jQuery(function() {
             "nav-subject-indexing": "Sisällönkuvailu",
             "project": "Sanasto ja tekstin kieli",
             "limit": "Ehdotusten enimmäismäärä",
+            "label-language": "Termien kieli",
+            "label-language-option-project": "projekti",
```
Comment (osma) [inline ai.finto.fi/static/js/i18n.js]: "tekstin kieli"

**Review by osma — CHANGES_REQUESTED** [review]
Looks good! Some suggestions for the translations below

**Comment by osma** [comment]
> At least this case remains to be solved: if there is no term for a concept in a language, like for Pohjanmaa in English, that label is not shown in the list of suggestions (or an empty string is shown):

Maybe you could do the same that Finto/Skosmos does in similar situations: use the untranslated value, with the language in parentheses. So this would become `Pohjanmaa (fi)`

**Pull request merged**